android/app/build.gradle

```diff
@@ -199,8 +190,23 @@ android {
         pickFirst '**/x86_64/libc++_shared.so'
         pickFirst '**/x86/libjsc.so'
         pickFirst '**/armeabi-v7a/libjsc.so'
+        // exclude
+        exclude 'META-INF/DEPENDENCIES'
+        exclude 'META-INF/LICENSE'
+        exclude 'META-INF/LICENSE.txt'
+        exclude 'META-INF/license.txt'
+        exclude 'META-INF/NOTICE'
+        exclude 'META-INF/NOTICE.txt'
+        exclude 'META-INF/notice.txt'
+        exclude 'META-INF/ASL2.0'
+        exclude("META-INF/*.kotlin_module")
     }
+}
 
+configurations.all {
+    resolutionStrategy {
+        force 'com.google.guava:guava:31.1-android'
```

Comment:
Do we still need this if you already made the change on the other repo?

Reply:
probably not, no!